salt/modules/consul.py

```diff
@@ -2126,7 +2131,7 @@ def acl_delete(consul_url=None, **kwargs):
         ret['res'] = False
         return ret
 
-    function = 'acl/delete/{0}'.format(kwargs['id'])
+    function = 'acl/destroy/{0}'.format(kwargs['id'])
```

Comment:
the problem is as they say here : https://www.consul.io/api/index.html#version-prefix
"Consul does not yet promise backwards compatibility even with the v1 prefix. We'll remove this warning when this policy changes. We expect to reach API stability by Consul 1.0."